swim/member.go

```diff
@@ -61,6 +61,18 @@ func (m Member) incarnation() int64 {
 	return m.Incarnation
 }
 
+func (m *Member) populateFromChange(c *Change) {
+	m.Address = c.Address
+	m.Incarnation = c.Incarnation
+	m.Status = c.Status
+}
+
+// checksumString fills a buffer that is passed with the contents that this node
+// needs to add to the checksum string.
+func (m Member) checksumString(b *bytes.Buffer) {
+	fmt.Fprintf(b, "%s%s%v", m.Address, m.Status, m.Incarnation)
```

Comment:
- since the formatting is just an append of the three fields, why not write directly to the buffer?
- `b` could be an `io.Writer`